src/scheduler/resv_info.c

```diff
@@ -749,6 +750,7 @@ new_resv_info()
 	rinfo->execvnodes_seq = NULL;
 	rinfo->count = 0;
 	rinfo->is_standing = 0;
+	rinfo->check_alternate_nodes = 0;
```

Comment:
You need to copy this in dup_resv_info() as well.  If you don't, check_alternate_nodes will be 0 regardless of what the original is.

Reply:
Added.